src/DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -87,7 +88,7 @@ def __init__(self, *args, **kwargs):
         self.getOutput = False
         self.sendAccounting = True
         self.sendSubmissionAccounting = True
-
+        self.sendSubmissionMonitoring = True
```

Comment:
As a start I think it's better if this is by default `False`
```suggestion
        self.sendSubmissionMonitoring = False
```